### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by LaViel** [phonglk/slack-block-user-extension#4]
This extension suddenly stopped working on my browsers from today. Confirmed on Google Chrome and Vivaldi.
Any chance you could fix it to make it work again, please?

Edit: Thread Filter still works, but Message List Filter doesn't.

**Comment by phonglk** [comment]
I just checked. It works well for me

**Comment by LaViel** [comment]
Thank you for the response.
I did some more research and noticed that this issue is only present in one of the workplaces I'm in.

I created my own workplace. The extension works well in there, as the div classes of "c-message" and "c-message--adjacent" are present in DOM.
![image](https://i.imgur.com/zD9EW0j.png)

However, the other Slack workplace I participate in uses totally different div class names for some reason, causing the extension not to work.
![image](https://i.imgur.com/OuBx4oj.png)

I have no idea what's causing this and am struggling to make the extension work again. I would really appreciate it if you could help me out with this problem.

**Comment by phonglk** [comment]
Thank you for taking your time to help to investigate it. I also wonder that maybe this classname is specified in some workspace as well. There is some solution I can think of right now:

- Introduce the configuration to allow the user to modify that classname (default to `c-message`)
  - Pros: flexible, future proof
  - Cons: unnecessary complicated, user as you need to configure it properly to work
- Take your case into consideration as well (check for both classes)
  - Pros: quick, no need action from user
  - Cons: the codebase is branching, in the future, if there are more such case like that, the codebase will be a mess

I will go implement the 2nd one after I my work.

**Comment by LaViel** [comment]
Thank you very much for taking care of the issue.
I'd also like to report that if a blocked user/bot posts multiple messages in a row in a short period of time, only the first message gets blocked and the rest stay unhidden, making it look like this:
![image](https://i.imgur.com/nEWcGma.png)
I believe this occurs because the adjacent message containers in this workplace don't use the div class "c-message--adjacent".  Though there are some div classes that are exclusive to them such as "c-message_kit__actions--above" and "p-message_pane_message__compact_timestamp--adjacent".
Would it be possible to solve this issue by checking for one of these classes?

**Comment by LaViel** [comment]
Nevermind the last post, figured out I just had to edit the message list selector's destination and the class names a bit.
https://github.com/phonglk/slack-block-user-extension/compare/master...LaViel:patch-1 (But this only works in my case.  Not knowing programming I couldn't figure out how to make it work in both cases. I'd appreciate if you'd make it work in both cases when you have time to spare)

Also, another weird thing happened to me. The class names have been changed in all of the workplaces I'm a member of. None of the message containers in the workplaces now have the class names "c-message" or "c-message--adjacent", making the version 1.2.0 of this extension not work in any of them. 

I created yet another workplace, but this time the message containers' tags look like this.
`<div role="document" class="c-message_kit__background p-message_pane_message__message c-message_kit__message" data-qa="message_container" data-qa-unprocessed="false" data-qa-placeholder="false">`
And not like this
`<div class="c-message c-message--light" role="document" data-qa="message_container" data-qa-hover="true" data-qa-placeholder="false">`
I wonder if this change is happening just to me, or someone else as well.
